<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>direc_sii</class>
 <widget class="QDialog" name="direc_sii">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direcciones Web Service AEAT</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Facturas emitidas (PRUEBAS)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="emitidas_test_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Facturas recibidas (PRUEBAS)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="recibidas_test_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Facturas emitidas (REAL)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="emitidas_real_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Facturas recibidas (REAL)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="recibidas_real_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>xmlns:siiLR personalizado:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="sii_lr_lineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>xmlns:sii personalizado:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="sii_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelar_pushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptar_pushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>emitidas_test_lineEdit</tabstop>
  <tabstop>recibidas_test_lineEdit</tabstop>
  <tabstop>emitidas_real_lineEdit</tabstop>
  <tabstop>recibidas_real_lineEdit</tabstop>
  <tabstop>sii_lr_lineEdit</tabstop>
  <tabstop>sii_lineEdit</tabstop>
  <tabstop>cancelar_pushButton</tabstop>
  <tabstop>aceptar_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelar_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>direc_sii</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
